sympy/physics/control/lti.py

```diff
@@ -679,6 +683,52 @@ def __pow__(self, p):
 
     def __neg__(self):
         return TransferFunction(-self.num, self.den, self.var)
+    
+    
+    def bilinear(self):
+        """
+        Returns falling coeffs of H(z) from numerator and denominator.
+        H(z) is the corresponding discretized tf, discretized with 
+        bilinear transform method.
+        Coefficients are falling, i.e. H(z) = (az+b)/(cz+d) is returned
+        as [a, b], [c, d].
+
+
+        Examples
+        ========
+
+        >>> from sympy.physics.control.lti import TransferFunction
+        >>> from sympy.abc import s, L, R, T, z
+        >>> tf = TransferFunction(1, s*L + R, s)
+        >>> numZ, denZ = tf.bilinear()
+        >>> numZ
+        [T, T]
+        >>> denZ
+        [2*L + R*T, -2*L + R*T]
+        """        
+        
+        z = Symbol('z') # discrete variable z 
+        T = Symbol('T')  # and sample period T
+       
+        H = self.num/self.den
```

Comment:
One workaround I see here which I would consider fundamentally incorrect is taking input in the form of a `TransferFunction` but then converting it into a symbolic expression and then working on it.

```
>>> from sympy import *
>>> from sympy.physics.control.lti import TransferFunction
>>> from sympy.abc import s, L, R, T, z
>>> tf = TransferFunction(1, s*L + R, s)
>>> type(tf)
<class 'sympy.physics.control.lti.TransferFunction'>
>>> H = tf.num/tf.den
>>> type(H)
<class 'sympy.core.power.Pow'>
```

This is not incorrect, but we are aware that `subs`, `collect`, `expand` do not work with the 'TransferFunction' class as of now. I would prefer that to be modified and then the entire function to be working on a variable of type - `<class 'sympy.physics.control.lti.TransferFunction'>`
Basically, if we are changing something then let's to it fully right. But if others agree then we can proceed with this too (it is clearly a useful addition).

Reply:
@faze-geek , That proposal is realized in commit

[https://github.com/sympy/sympy/commit/36d6d4a608b6a991e2dc5acc31af6b89e11aaf85](https://github.com/sympy/sympy/commit/36d6d4a608b6a991e2dc5acc31af6b89e11aaf85)

thanks to comments, suggestions and contributions of @sylee957 and @hanspi42

Reply:
Thanks @zolabar , looks good to me. You can mark this conversation resolved accordingly.